### PR TITLE
Fix conditional sampling for tabular presets

### LINKED ISSUE #796: Unable to sample conditionally in Tabular_Preset model
### Environment Details

Please indicate the following details about the environment in which you found the bug:

* SDV version: 0.14.1
* Python version: 3.8.10
* Operating System: Ubuntu 20.04.3 

### Error Description

I am just reproducing this code  at https://sdv.dev/SDV/user_guides/single_table/tabular_preset.html#conditional-sampling. 

### Steps to reproduce
Just run everything on the page till https://sdv.dev/SDV/user_guides/single_table/tabular_preset.html#conditional-sampling 

```
If there was a crash, please include the traceback here.
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-16-5e56e35112b9> in <module>
----> 1 model.sample_conditions(conditions = all_conditions)

~/.local/lib/python3.8/site-packages/sdv/lite/tabular.py in sample_conditions(self, conditions, max_tries, batch_size_per_try, randomize_samples, output_file_path)
    196                 Sampled data.
    197         """
--> 198         sampled = self._model.sample_conditions(
    199             conditions, max_tries, batch_size_per_try, randomize_samples, output_file_path)
    200 

TypeError: sample_conditions() takes from 2 to 5 positional arguments but 6 were given

### PR DESCRIPTION
**Pull request by katxiao**
Resolves #796

### CODE REVIEW

**Review by pvk-developer — APPROVED** [review]
LGTM!

File: sdv/lite/tabular.py [hunk 1]
```diff
@@ -251,7 +272,11 @@ def load(cls, path):
             TabularModel:
                 The loaded tabular model.
         """
-        return cls._default_model.load(path)
+        with open(path, 'rb') as f:
+            model = pickle.load(f)
+            throw_version_mismatch_warning(getattr(model, '_package_versions', None))
+
+            return
```
Comment (amontanez24) [inline sdv/lite/tabular.py:279]: Should this return the model?

Comment (katxiao) [reply sdv/lite/tabular.py]: Yes, good catch. I must've deleted it by accident.. updated the unit test as well to test this.

**Review by amontanez24 — APPROVED** [review]
LGTM!

**Pull request merged**